.github/workflows/vettools/vettools.go

```diff
@@ -18,10 +18,28 @@
 package main
 
 import (
+	"os"
+	"runtime"
+
+	"github.com/kisielk/errcheck/errcheck"
 	"golang.org/x/tools/go/analysis/multichecker"
 	"golang.org/x/tools/go/analysis/passes/atomicalign"
 )
 
 func main() {
-	multichecker.Main(atomicalign.Analyzer)
+	GOOS, ok := os.LookupEnv("GOOS")
+	if !ok {
+		GOOS = runtime.GOOS
+	}
+
+	for _, filename := range []string{
+		"errcheck-excludes.txt",
+		"errcheck-excludes-" + GOOS + ".txt",
```

Comment:
I prefer `_` to `-`

Reply:
Can we use the name with `.` like these?

`.errcheck_excludes`
`.errcheck_excludes_windows`

Reply:
yes :+1: